src/app/config/Sample_Excel_Word.ts

```diff
@@ -0,0 +1,176 @@
+const shell = require('shelljs');
+const ProgressBar = require('progress');
+const log = require('single-line-log').stdout;
+const Spinner = require('cli-spinner').Spinner;
+const { spawn } = require('child_process');
+const { exec } = require('child_process');
+const { execSync } = require('child_process');
+const readline = require('readline');
+
+import * as os from 'os';
+
+
+shell.config.silent = true;
+
+async function exec_script_Excel_Mail(){
+  // shell.exec('code .');
+  console.log('Welcome to experience this Office add-in sample: Excel Mail merge Add-in project!');
+
+  return new Promise<boolean>((resolve, reject) => {
+    
+        let is_vscode_installed = false;
+
+        console.log('Welcome to experience this Office add-in sample!');
+
+        // Step 1: Get sample code
+        console.log('Step [1/3]: Getting sample code...');
+        let spinner = new Spinner('Processing.. %s');
+        spinner.setSpinnerString('|/-\\');
+        spinner.start();
+
+        shell.exec('git clone https://github.com/OfficeDev/Excel-Scenario-based-Add-in-Samples.git', {async:true}, (code, stdout, stderr) => {
```

Comment:
By cloning the project you are keeping a user connection with the original template/sample repo . . . which is not what we're trying to do here.  We're creating a new project that is not connected back to a template repo.  We also already have code in this plugin that accomplishes that . . . this is taking a completely divergent path.